server/config.go

```diff
@@ -289,14 +299,26 @@ func (c *config) GetLeaderboard() *LeaderboardConfig {
 	return c.Leaderboard
 }
 
+// ShutdownConfig is configuration relevant to server shutdown behaviour.
+type ShutdownConfig struct {
+	GracePeriodSec int `yaml:"grace_period_sec" json:"grace_period_sec" usage:"Maximum number of seconds to wait for the server to complete work before shutting down. Default is 60 seconds. If 0 the server will shut down immediately when it receives a termination singal."`
+}
+
+// NewShutdownConfig creates a new ShutdownConfig struct.
+func NewShutdownConfig() *ShutdownConfig {
+	return &ShutdownConfig{
+		GracePeriodSec: 60,
```

Comment:
I wonder whether we should keep it as 0 for backwards compatibility?

Reply:
I vote for `0` too.